src/app/components/StartWindow.js

```diff
@@ -0,0 +1,82 @@
+export const StartWindow = (callBackFunction) => {
+  const renderedComponent = document.createElement('section');
+  renderedComponent.classList.add('startWindow');
+
+  const deathStarImageComponent = document.createElement('img');
+  deathStarImageComponent.setAttribute(
+    'src',
+    '../../static/assets/ui/DeathStar.png',
+  );
+  deathStarImageComponent.classList.add('startWindow__deathStar');
+
+  const startGameInformation = document.createElement('span');
+  startGameInformation.classList.add('startWindow__startGameInformation');
+  startGameInformation.innerHTML = 'Click <br> to start the game!';
+
+  const deathStarContainer = document.createElement('div');
+  deathStarContainer.classList.add('startWindow__deathStarContainer');
+
+  const fadeComponent = document.createElement('div');
+  fadeComponent.classList.add('startWindow__fadeComponent');
+
+  const trailerComponent = document.createElement('article');
+  trailerComponent.classList.add('startWindow__trailerComponent');
+  const crawlElement = document.createElement('div');
+  crawlElement.classList.add('startWindow__crawlElement');
+  const header = document.createElement('div');
+  header.classList.add('startWindow__header');
+  const episodeSubtitle = document.createElement('h1');
+  episodeSubtitle.classList.add('startWindow__episodeSubtitle');
+  episodeSubtitle.textContent = 'Episode IV';
+  const episodeTitle = document.createElement('p');
+  episodeTitle.classList.add('startWindow__episodeTitle');
+  // ? episodeTitle.textContent = 'A New Hope';
+  episodeTitle.textContent = 'Coders Camp cereals';
```

Comment:
Niezbyt rozumiem znaczenie słowa "cereals" tutaj - ale ogólnie mi się podoba :) 
Tego raczej nie trzeba "upiększać". 

Powiedzcie czy Wy to sami wymyśliliście, czy posiłkowaliście się jakimiś przykładami z internetu (i tak to przecież codzienność developera -więc dobrze)!

Pamiętam kiedyś opowieść o projekcie, gdzie była jakaś nowa strona firmy i frontendowcy poświęcili czas na zaprogramowanie "krutyny", za którą była treść strony, żeby prezes mógł sobie nacisnąć i się podniosła :P